### DISCUSSION TITLE
Type Error when running

### DISCUSSION THREAD

**Discussion by MertUzeken** [imartinez/privateGPT#782] [General]
Hello,

I'm trying to implement GPT4All into my existing project.
While running I get following error:

`Traceback (most recent call last):
  File "/home/sadbox/Desktop/LangChain/gpt4all.py", line 115, in <module>
    llm = GPT4All(model=r"/home/sadbox/Desktop/LangChain/models/gpt4.bin", n_ctx= model_n_ctx, callbacks=callbacks)
  File "/home/sadbox/.local/lib/python3.10/site-packages/langchain/load/serializable.py", line 61, in __init__
    super().__init__(**kwargs)
  File "pydantic/main.py", line 341, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for GPT4All
__root__ -> __root__Serializable.__init__() takes 1 positional argument but 2 were given (type=type_error)`
  
  
My Code:
```
embeddings = HuggingFaceEmbeddings(model_name=EMBEDDINGS_MODEL_NAME)

db = Chroma(persist_directory=persist_directory,
embedding_function=embeddings, 
client_settings=Settings(chroma_db_impl='duckdb+parquet',

                     persist_directory=persist_directory,

                     anonymized_telemetry=False))

callbacks = [StreamingStdOutCallbackHandler()]
retriever = db.as_retriever(search_kwargs={"k":target_source_chunks})

llm = GPT4All(model=r"/home/sadbox/Desktop/LangChain/models/gpt4.bin", n_ctx= model_n_ctx, callbacks=callbacks)

pdf_qa = RetrievalQA.from_chain_type(llm=llm, chain_type="stuff", retriever=retriever)

chat_history = []
```

It seems like the error comes from a library which is a dependency for gpt4all, but since I'm just a beginner, I cant solve it.
Note: I tried a lot of different solutions, even the ones found here and none seem to work.

Thanks in advance

**Comment by mzamini92** [comment]
The error message you are getting is telling you that the __root__Serializable class in the langchain.load.serializable module is expecting only one positional argument, but you are passing it two. have you tried this?
```
llm = GPT4All(model=r"/home/sadbox/Desktop/LangChain/models/gpt4.bin")
```
